src/single-spa-examples.js

```diff
@@ -10,46 +10,58 @@ singleSpa.declareChildApplication('svelte', () => SystemJS.import('/build/svelte
 singleSpa.declareChildApplication('preact', () => SystemJS.import('/build/preact.app.js'), hashPrefix('/preact'));
 singleSpa.declareChildApplication('iframe-vanilla-js', () => SystemJS.import('/build/vanilla.app.js'), hashPrefix('/vanilla'));
 singleSpa.declareChildApplication('inferno', () => SystemJS.import('/build/inferno.app.js'), hashPrefix('/inferno'));
-singleSpa.declareChildApplication('ember', () => loadEmberScript('/build/ember-app/assets/ember-app.js'), hashPrefix('/ember'));
+singleSpa.declareChildApplication('ember', () => loadEmberScript("ember-app"), hashPrefix('/ember'));
```

Comment:
passing app-name instead so that we can load multiple ember apps in the same page